tests/tasks/airbyte/test_airbyte.py

```diff
@@ -21,6 +21,13 @@ def test_connection_id_must_be_provided(self):
         with pytest.raises(ValueError):
             task.run()
 
+    def test_optional_params_are_optional(self):
+        task = AirbyteConnectionTask(
+            connection_id="749c19dc-4f97-4f30-bb0f-126e53506960"
+        )
+        with pytest.raises(AirbyteServerNotHealthyException):
+            task.run()
```

Comment:
Thanks for adding a test to make sure that this functionality is covered! I think this test is coupled to the internals of the task, so if we needed to change the name of `AirbyteServerNotHealthyException`, this test would fail when it shouldn't. Instead, I think we should test that a `ValueError` is not raised. Here's a suggestion of how we could do that:
```suggestion
    def test_optional_params_are_optional(self):
        task = AirbyteConnectionTask(
            connection_id="749c19dc-4f97-4f30-bb0f-126e53506960"
        )
        try:
            task.run()
        except ValueError as err:
            assert False, str(err)
        except Exception:
            pass
```

Reply:
Good catch, I'll add your changes. Thanks!

Reply:
@desertaxle done